packages/aztec.js/test/proof/bilateralSwap/verifier.spec.js

```diff
@@ -67,73 +65,75 @@ describe('AZTEC bilateral swap verifier tests', () => {
     describe('failure states', () => {
         let testNotes;
         let sender;
+        let parseInputs;
 
         beforeEach(() => {
-            testNotes = helpers.makeTestNotes([10, 20], [10, 20]);
+            // to test failure states we need to pass in bad data to verifier
+            // so we need to turn off proof.parseInputs
+            parseInputs = sinon.stub(proofUtils, 'parseInputs').callsFake(() => { });
+
+            testNotes = proofUtils.makeTestNotes([10, 20], [10, 20]);
```

Comment:
I think it makes sense to move this to the first test to align with structure of the other tests.

Reply:
Done